src/framework/components/anim/component.js

```diff
@@ -87,7 +87,7 @@ class AnimComponent extends Component {
             var layerName = layer.name;
             for (var j = 0; j < layer.states.length; j++) {
                 var stateName = layer.states[j];
-                if (stateName !== ANIM_STATE_START && stateName !== ANIM_STATE_END && stateName !== ANIM_STATE_ANY) {
+                if (!ANIM_CONTROL_STATES.includes(stateName)) {

```

Comment:
Not sure what happened to my previous comment, but `Array#includes` doesn't exist in IE11. Have you checked the Babel output to see if it's IE11-compatible?